### PR TITLE
Base64 de/encoding

### PR DESCRIPTION
**Pull request by semantosoph**
Hi.

I implemented a simple Base64 de/encoder (with tests) and used it to enable the CcBase64 codec and its tests. This is only the foundation for more thing to come, but these changes work for themselves and could thus be pulled separately. Also, bumped the version in pom.xml.

Regards, Sven

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this pull request

**Comment by yegor256** [comment]
@semantosoph looks cool, but you have to run `mvn clean install -Pqulice` and fix the problems qulice reports

**Comment by semantosoph** [comment]
@yegor256 Done that. Qulice now runs error free for me.

File: src/test/java/org/takes/misc/Base64Test.java [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+ * @version $Id$
+ * @since 1.1
+ */
+@Ignore
+public final class Base64Test {
+
+    /**
+     * A short text to encode with Base64.
+     */
+    private static final String SHORTTEXT =
```
Comment (yegor256) [inline src/test/java/org/takes/misc/Base64Test.java]: @semantosoph check this out: http://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html

Comment (semantosoph) [inline src/test/java/org/takes/misc/Base64Test.java]: @yegor256 I see your point and refactored the class.

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by semantosoph** [comment]
Oops: "Error: M2_HOME is set to an invalid directory."

Is there something I can do?

**Comment by semantosoph** [comment]
@yegor256 Could you fix the appveyor (or turn it off), please? I'd like to move on here.

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by yegor256** [comment]
@semantosoph thanks!

**Pull request merged**